### PR TITLE
{bio}[foss/2022a] Stacks v2.62

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bdw-opensuse-154 - Linux openSUSE Leap 15.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.15
See https://gist.github.com/9dda41f409aad14b2b5ee04d5f05cc8d for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @sassy-crick!

File: easybuild/easyconfigs/s/Stacks/Stacks-2.62-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description = """Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
+ the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,
+ for the purpose of building genetic maps and conducting population genomics and phylogeography.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://catchenlab.life.illinois.edu/stacks/source/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['01c814588fd72c13494785af49123a6a2c20495c924ed84965fdd4e0da1ab524']
```
Comment (boegel) [inline easybuild/easyconfigs/s/Stacks/Stacks-2.62-foss-2022a.eb:16]: FYI: there was a silent re-release of Stacks v2.62, so checksum needs to be fixed, see #16134

Comment (sassy-crick) [reply easybuild/easyconfigs/s/Stacks/Stacks-2.62-foss-2022a.eb:16]: Thanks for doing this. These things are a nuisance! I have rebuild and uploaded the test-report for PR #16134

**Pull request merged**